services/tasks-svc/internal/ward/ward.go

```diff
@@ -66,14 +110,15 @@ func (ServiceServer) GetWard(ctx context.Context, req *pb.GetWardRequest) (*pb.G
 		return nil, err
 	}
 
+	// return
```

Comment:
```suggestion
```

Reply:
nah, these comments help with fragmenting the code into readable sub-blocks
the return comment ends the "do query" part

I think it makes the code more readable

![image](https://github.com/user-attachments/assets/46b1e523-faff-4e8a-aa40-ac541b76c229)

Reply:
```golang
// return
return &pb.GetWardResponse{
	Id:          ward.ID.String(),
	Name:        ward.Name,
	Consistency: common.ConsistencyToken(ward.Consistency).String(), //nolint:gosec
}, nil
```

I understand your reasoning. I only mentioned the `return` because I find a `return` comment before the actual return quite redundant. LOCs where we do larger preps for the return payload, I didnt mentioned it.